[2024-05-23T08:45:17.968+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-23T08:45:17.987+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag_parallel.group_a.transform_load_weather_data manual__2024-05-23T08:44:25.689117+00:00 [queued]>
[2024-05-23T08:45:18.005+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag_parallel.group_a.transform_load_weather_data manual__2024-05-23T08:44:25.689117+00:00 [queued]>
[2024-05-23T08:45:18.008+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-05-23T08:45:18.052+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): group_a.transform_load_weather_data> on 2024-05-23 08:44:25.689117+00:00
[2024-05-23T08:45:18.061+0000] {standard_task_runner.py:63} INFO - Started process 25048 to run task
[2024-05-23T08:45:18.067+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag_parallel', 'group_a.transform_load_weather_data', 'manual__2024-05-23T08:44:25.689117+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/weatherapi_dag.py', '--cfg-path', '/tmp/tmpmkcjxkp8']
[2024-05-23T08:45:18.070+0000] {standard_task_runner.py:91} INFO - Job 117: Subtask group_a.transform_load_weather_data
[2024-05-23T08:45:18.164+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag_parallel.group_a.transform_load_weather_data manual__2024-05-23T08:44:25.689117+00:00 [running]> on host ubuntu2204
[2024-05-23T08:45:18.335+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag_parallel' AIRFLOW_CTX_TASK_ID='group_a.transform_load_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-23T08:44:25.689117+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-23T08:44:25.689117+00:00'
[2024-05-23T08:45:18.338+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-23T08:45:19.169+0000] {credentials.py:621} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2024-05-23T08:45:21.890+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-23T08:45:21.892+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-23T08:45:21.899+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag_parallel, task_id=group_a.transform_load_weather_data, run_id=manual__2024-05-23T08:44:25.689117+00:00, execution_date=20240523T084425, start_date=20240523T084517, end_date=20240523T084521
[2024-05-23T08:45:21.968+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-23T08:45:21.988+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-23T08:45:21.990+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
